robottelo.properties.sample

```diff
@@ -122,8 +122,11 @@ ssh_key=
 
 # Fake manifest testing.
 # [fake_manifest]
```

Comment:
this is now a list of key=value pairs and is being parsed by dict cast function, which was already in place